core/src/main/java/tech/pegasys/eth2signer/core/multikey/metadata/AzureSecretSigningMetadata.java

```diff
@@ -31,7 +32,9 @@ public AzureSecretSigningMetadata(
       @JsonProperty("clientSecret") final String clientSecret,
       @JsonProperty("tenantId") final String tenantId,
       @JsonProperty("vaultName") final String vaultName,
-      @JsonProperty("secretName") final String secretName) {
+      @JsonProperty("secretName") final String secretName,
+      @JsonProperty(value = "keyType") final KeyType keyType) {
```

Comment:
to be consistent with other @JsonProperty annotations can use leave out the value = and just use @JsonProperty("keyType"). Same for the other SigningMetadata classes

Reply:
for sure - oops.